Formula/llvm.rb

```diff
@@ -59,15 +59,15 @@ def install
       clang-tools-extra
       lld
       lldb
-      openmp
-      polly
       mlir
+      polly
     ]
     runtimes = %w[
       compiler-rt
       libcxx
       libcxxabi
       libunwind
+      openmp
```

Comment:
Couldn't get `openmp` to build on ARM while in the `projects` array. It doesn't seem like this should have negative consequences... Though we may want to restrict this change to macOS.

Reply:
Actually, never mind. It shouldn't matter on Linux either, as it gets built with Clang whether it's in `projects` or `runtimes`. It's just a difference on which Clang you use to build it.

Reply:
I suspect the change is due to my limited choice of `LLVM_TARGETS_TO_BUILD` for bootstrapping since the build fails with a complaint about missing symbols for `arm64` on ARM.